app/forms/donation_form.rb

```diff
@@ -1,5 +1,5 @@
 class DonationForm < Form
-  DATETIME_FORMAT = '%Y-%m-%d %I:%M %p'
+  DATETIME_FORMAT = '%m/%d/%Y %I:%M %p'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.